js/script.js

```diff
@@ -93,17 +93,17 @@ function getRandomQuote () {
   return quotes[getRandomQuote];
 }
 console.log(getRandomQuote());
-
 /***
  * `printQuote` function
 ***/
 function printQuote () {
   let message = "";
   let showQuote = getRandomQuote(quotes);
```

Comment:
to avoid having a extra variable called showQuote here....you can 'restructure' the object keys like this: `const { quote, source, site,year} = getRandomQuote(quotes)` , then you can use quote, source, etc as variables....you can learn about destructuring [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)